src/components/pacbio/PacbioLibraryCreate.vue

```diff
@@ -22,10 +22,20 @@
           {{ selectedSample.sample_name }} ({{ selectedSample.source_identifier }})
         </traction-form-group>
 
-        <traction-form-group id="tag-select-input" label="Tag:" label-for="tag-input">
+        <traction-form-group id="tag-set-select-input" label-for="tag-set-input" label="Tag:">
+          <traction-select
+            id="tag-set-input"
+            v-model="selectedTagSetId"
+            data-type="tag-set-list"
+            :options="tagSetOptions"
+            @input="resetSelectedTagId"
+          ></traction-select>
+        </traction-form-group>
+
+        <traction-form-group id="tag-select-input" label-for="tag-input">
```

Comment:
Would be nice to have this conditionally disabled based on if a tag set has been selected

Reply:
If we disable it, they can't change it again before submitting.

Reply:
Im not sure what you mean. Conditionally disabling the tag select based on whether a tagSet has been selected should be fine?

Reply:
Just to clarify what I think @BenTopping is saying.  If they haven't selected a tag set in the drop down above, then the specific tag drop down this comment is against should be disabled as it makes no sense to select a tag without selecting a tag set.  Then when a tag set has been selected, the specific tag drop down should become enabled.

Reply:
Now the second dropdown (tag) is enabled if there is a selection in the first (tagset). It will switch between enabled/disabled accordingly.